azure-pipelines.yml

```diff
@@ -128,4 +128,5 @@ stages:
       inputs:
         testResultsFormat: 'XUnit'
         testREsultsFiles: '**/testresults.xunit.xml'
```

Comment:
Obviously not related to your change, but it looks like there is a case typo here. Might not matter if its case-insensitive.

Reply:
lol... I'll fix that too